Formula/kubernetes-cli.rb

```diff
@@ -30,7 +30,7 @@ def install
 
     dir = "_output/local/bin/darwin/#{arch}"
     bin.install "#{dir}/kubectl"
-    bash_completion.install "contrib/completions/bash/kubectl"
+    (bash_completion/"kubectl").write `#{bin}/kubectl completion bash`
```

Comment:
I believe `completion` subcommand was implemenented in v1.3.0-alpha5. I confirmed `completion` works in v1.3.0 (stable) and 1.4.0-alpha.0 (devel).

https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md/#v130-alpha5
> * Move shell completion generation into 'kubectl completion' command ([#23801](https://github.com/kubernetes/kubernetes/pull/23801), [@sttts](https://github.com/sttts))